src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Request as a Service",
+    "version": "4.0",
+    "title": "Concur Request API Documentation"
+  },
+  "host": "www.concursolutions.com",
+  "basePath": "/travelrequest/v4",
+  "tags": [
+    {
+      "name": "Request Resource",
+      "description": "Manage documents used for pre-spend authorizations within Concur Request"
+    },
+    {
+      "name": "Workflow Resource",
+      "description": "Manage workflow transitions for a Request document"
+    },
+    {
+      "name": "Travel Agency Resource",
+      "description": "Manage the configuration for travel agencies integrated with Concur Request"
+    },
+    {
+      "name": "Policy Resource",
+      "description": "Manage policy settings"
```

Comment:
We just offer to read policy ids, therefore it should be something like "Get available Request Policies"

Reply:
1/ Rename Policy Resource by Request Policy Resource
2/ Ok for description